### PR TITLE
EZP-31541: Added new CACHE_NAMESPACE env to redis docker stack

### PR DESCRIPTION
**Pull request by ViniTou**
https://jira.ez.no/browse/EZP-31541
followup to: https://github.com/ezsystems/ezplatform/pull/527

### CODE REVIEW

File: doc/docker/redis.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ services:
     environment:
      - CACHE_POOL=cache.redis
      - CACHE_DSN=redis:6379
+     - CACHE_NAMESPACE
```
Comment (michal-myszka) [inline doc/docker/redis.yml:15]: Maybe default namespace name should be provided here?

Comment (ViniTou) [reply doc/docker/redis.yml:15]: It is in .env, not sure if we should duplicate that

Comment (mnocon) [reply doc/docker/redis.yml:15]: For me `.env` is enough

**Comment by lserwatka** [comment]
@ViniTou could you merge it to EE?

**Pull request merged**